controls/V-71999.rb

```diff
@@ -55,14 +55,16 @@
   tag "fix": "Install the operating system patches or updated packages
 available from Red Hat within 30 days or sooner as local policy dictates."
   tag "fix_id": "F-78351r1_fix"
-  describe.one
-    describe "All packages are up to date" do
+  describe.one do
+    describe 'List of out-of-date packages' do
       subject { linux_update.updates.length }
-      it { should eq 0 }
+      it { should be_empty }
```

Comment:
Are you sure this is correct?

`length` returns an `Integer`.

You could remove `.length` from the `subject` line and I think it would be correct.